dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -40,6 +39,8 @@
  */
 public class WorkflowHelper {
 
+    private static final Set<WorkflowStatus> DEFAULT_SHOW_ON = CollectionsUtils.set(WorkflowStatus.LOCKED, WorkflowStatus.UNLOCKED);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Set to an EnumSet. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1641)

Reply:
DONE